### PR TITLE
Capinfo integration.

### PR DESCRIPTION
**Pull request by jacobbramley**
Proposed integration of Capinfo.

* Everything under `assets/capinfo/` comes from the output of [Capinfo](https://git.morello-project.org/morello/utilities/capinfo/) ([2e07a86](https://git.morello-project.org/morello/utilities/capinfo/-/commit/2e07a86e07c0558383244ce74a499337c6325294))
* As discussed, I had some trouble getting the ES modules and Wasm loader to work with Jekyll. My approach works, but feels like a hack. If you know of alternative solutions, I'm open to suggestions.
* The licence presentation is under discussion; please don't accept this PR until that thread is resolved.

Screenshot

### CODE REVIEW

**Comment by bfletcher** [comment]
Hi Jacob, looks great. Would it be possible to add a link to the capinfo source repo as well as the licence?

**Comment by jacobbramley** [comment]
Updated with a link, as requested.
Screenshot 2021-08-11 at 15 07 22

File: _posts/resources/2021-06-28-capinfo.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+author: arm
+title: Capinfo
+date: 2020-11-26 14:00:00
```
Comment (jacobbramley) [inline _posts/resources/2021-06-28-capinfo.md:4]: We probably need to bump this date (when the PR is otherwise ready to merge).

**Comment by jacobbramley** [comment]
Thanks!

**Comment by kylekirkby** [comment]
> Thanks!

Looks awesome! Cheers @jacobbramley!

**Pull request merged**